locales/mobileLocales/en.js

```diff
@@ -0,0 +1,33 @@
+export const en = {
+  capture: {
+    driving_licence: {
+      front: {
+        instructions: 'Take a photo of the front of your license',
+        webcam: ''
```

Comment:
Minor: do we need it? the component shouldn't ask for strings if he doesn't want to display them